pageserver/src/basebackup.rs

```diff
@@ -161,7 +161,7 @@ fn add_pgcontrol_file(
             // TODO: When we restart master there are no active transaction and oldestXid is
             // equal to nextXid if there are no prepared transactions.
             // Let's ignore them for a while...
-            checkpoint.oldestXid = checkpoint.nextXid.value as u32;
```

Comment:
This comment seems misleading. Let's update it too
@knizhnik  Did you mean some other field here?  I assume it can relate to checkpoint.oldestActiveXid ?